akka/templates/deployment.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
     {{- end }}
   name: "{{ include "akka.fullname" . }}"
 spec:
-  {{- if and (not .Values.autoscaling.enabled) .Values.replicaCount }}
+  {{- if and (not .Values.autoscaling.enabled) (ne .Values.replicaCount nil) }}
```

Comment:
Change `.Values.replicaCount` to `ne .Values.replicaCount nil` because we want to allow a setting of 0, even though it currently becomes `true` when `.Values.replicaCount` is not `nil` or `0`.